### ISSUE TITLE
extra options error message.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halamalala/inkscape2tikz#24]
```
What steps will reproduce the problem?
1.Open the file book-exercise-icon.svg(the file is attached.) in inkscape.
2.Choose effects->export->export to tikz path.
3.Choose the options shown in the screenshot.jpeg

What is the expected output? What do you see instead?
Expected a tex file with tikz code, but got an error message.

What version of the product are you using? On what operating system?
Ubuntu 9.04, inkscape 4.6

Please provide any additional information below.
Here is the traceback:

Traceback (most recent call last):
  File "/usr/share/inkscape/extensions/tikz_export.py", line 1304, in <module>
    main_inkscape()
  File "/usr/share/inkscape/extensions/tikz_export.py", line 1294, in
main_inkscape
    effect.affect()
  File "/usr/share/inkscape/extensions/inkex.py", line 154, in affect
    self.effect()
  File "/usr/share/inkscape/extensions/tikz_export.py", line 1231, in effect
    output = FIG_TEMPLATE % dict(pathcode=s, colorcode=self.colorcode)
KeyError: 'extraoptions'


The screenshot file is also attached.

```

Original issue reported on code.google.com by `venkat.s...@gmail.com` on 4 Feb 2010 at 8:07

Attachments:
- [book-exercise-icon.svg](https://storage.googleapis.com/google-code-attachments/inkscape2tikz/issue-24/comment-0/book-exercise-icon.svg)
- [screenshot.jpg](https://storage.googleapis.com/google-code-attachments/inkscape2tikz/issue-24/comment-0/screenshot.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for spotting this. I will fix this error soon. Until then, select 'Stand
alone' output to avoid raising the exception.
```

Original comment by `kjel...@gmail.com` on 4 Feb 2010 at 8:14
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 1ac3c2a834.
```

Original comment by `kjel...@gmail.com` on 4 Feb 2010 at 5:14
- Changed state: **Fixed**
